sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/InjectRuntimeFilter.scala

```diff
@@ -185,14 +186,18 @@ object InjectRuntimeFilter extends Rule[LogicalPlan] with PredicateHelper with J
   }
 
   private def isProbablyShuffleJoin(left: LogicalPlan,
-      right: LogicalPlan, hint: JoinHint): Boolean = {
-    !hintToBroadcastLeft(hint) && !hintToBroadcastRight(hint) &&
-      !canBroadcastBySize(left, conf) && !canBroadcastBySize(right, conf)
+                                    right: LogicalPlan,
+                                    hint: JoinHint,
+                                    joinType: JoinType): Boolean = {
```

Comment:
Could you please indent this better, use 4 spaces starting from the function definition for every parameter on new line

Reply:
done

> Could you please indent this better, use 4 spaces starting from the function definition for every parameter on new line

done